pyseir/inference/model_fitter.py

```diff
@@ -109,6 +107,8 @@ class ModelFitter:
         ),
     }
 
+    REFF_LOWER_BOUND = 0.7
```

Comment:
Why do you think this is a good lower bound? Not that I have a better idea, just want to check the reasoning here.